mage_ai/frontend/components/ServerTimeDropdown/utils.ts

```diff
@@ -0,0 +1,34 @@
+import { 
+  LOCAL_STORAGE_KEY_DISPLAY_LOCAL_SERVER_TIME, LOCAL_STORAGE_KEY_INCLUDE_SERVER_TIME_SECONDS, 
+} from '@storage/constants';
+import { get, set } from '@storage/localStorage';
+
+function getLocalStorageValue(storageKey: string): boolean {
+  const storageValue = get(storageKey, null);
+
+  return storageValue || false;
+}
+
+function setLocalStorageValue(storageKey: string, value: boolean): boolean {
```

Comment:
We could possibly move this method to the `mage_ai/frontend/storage/localStorage.ts` file, since it seems more generalized and something we could use again in the future.